Sources/PaymentButtons/PaymentButton.swift

```diff
@@ -256,6 +257,16 @@ public class PaymentButton: UIButton {
         }
     }
 
+    public override var intrinsicContentSize: CGSize {
```

Comment:
Lets call out these size restrictions in the changelog as a bug fix

Reply:
Wasn't sure how to write it in the CHANGELOG - `Fixed button alignment and size bug in SwiftUI with `intrinsicContentSize``